system-addon/lib/NewTabInit.jsm

```diff
@@ -15,7 +15,7 @@ this.NewTabInit = class NewTabInit {
   onAction(action) {
     let newAction;
     switch (action.type) {
-      case at.NEW_TAB_LOAD:
+      case at.NEW_TAB_STATE_REQUEST:
```

Comment:
This wants a unit test, I think.  Maybe a look at the code coverage numbers would show other changes that want one too?

Reply:
Currently this file isn't being tested (which is an oversight), do you mind if I file a follow-up for this?

Reply:
Because of the high-value in getting this into the tree sooner rather than later, yes.